src/components/feature.mjs

```diff
@@ -0,0 +1,83 @@
+import { parseExample }  from '../../utils/feature-example.js';
+import { backTicksToCodeTags, escapeHTML, html } from '../util/html.mjs';
+import { renderCaniuseFeature } from './caniuse-feature.mjs';
+import { renderFeaturePolyfills } from './feature-polyfills.mjs';
+
+const stages = [
+	'stage-0-aspirational',
+	'stage-1-experimental',
+	'stage-2-allowable',
+	'stage-3-embraced',
+	'stage-4-standardized',
+]
+
+export function renderFeature(feature) {
+
+	const imageName = `/images/stages/stage-${feature.stage}.svg`;
+	const badge = `/images/badges/${feature.id}.svg`;
+
+	const cleanTitle = feature.title.replace(/`/g, '');
+
+	return html`
+		<article class="cssdb-feature" id="${feature.id}">
+			<header class="cssdb-feature-header">
+				<a class="cssdb-feature-stage" href="#${stages[feature.stage]}">
+					<img class="cssdb-feature-stage-image" src="${imageName}" alt="" width="62" height="62" role="presentation" loading="lazy">
+				</a>
+				<h2 class="cssdb-feature-heading">
+					<a href="#${feature.id}">
+						${backTicksToCodeTags(escapeHTML(feature.title))}
+					</a>
+				</h2>
+				<p class="cssdb-feature-description">
+					${escapeHTML(feature.description)}
+				</p>
+			</header>
+			<p class="cssdb-feature-docs">
+				<a
+					href="${feature.specification}"
+					aria-label="Specification for ${escapeHTML(cleanTitle)}"
+				>
+					<img
+						src="https://img.shields.io/badge/Spec-005A9C?logo=w3c&style=flat-square"
+						alt="W3C Specification"
```

Comment:
Not sure what it was before but given the `aria-label` above I think this just adds noise? Should it just be empty so it's treated as decorative? 

Don't have a super-strong opinion here

Reply:
I agree.
All these labels were to verbose and contained too much repetition.
This definitely was annoying for screenreader users.

I've tried to clean this up a bit.